modules/react/popup/stories/Popup.stories.mdx

```diff
@@ -159,6 +159,12 @@ no UI and will render any children to the `body` element and position around a p
 Prefer using [Popup.Popper](#popuppopper) instead. Use this to make Popups that don't utilize a
 PopupModel or any associate popup [hooks](#hooks).
 
+> **Note:** `Popper` renders any children to a `div` element created by the `PopupStack`. This
+> element is not controlled by React, so any extra element props will _not_ be forwarded. The `ref`
+> will point to the `div` element created by the `PopupStack`, however. In v4, an extra `div`
+> element was rendered and that's where extra props were spread to. In v5+, you can provide your own
+> element if you wish.
+
```

Comment:
thanks for clarifying, i think this info kinda clears my issue i had open about spreading props on this element